src/internal/lock.h

```diff
@@ -3,7 +3,8 @@
 
 hidden void __lock(volatile int *);
 hidden void __unlock(volatile int *);
-#define LOCK(x) __lock(x)
-#define UNLOCK(x) __unlock(x)
+
+#define LOCK(x) (x)
+#define UNLOCK(x) (x)
```

Comment:
These can be implemented for the single threaded case, with the assumption that a double lock should not happen, we can assert that